distribution/ddf-catalog/pom.xml

```diff
@@ -156,7 +156,7 @@
                         <configuration>
                             <rules>
                                 <requireFilesSize>
-                                    <maxsize>310000000</maxsize>
+                                    <maxsize>311000000</maxsize>
```

Comment:
Why the bump here?

Reply:
DDF 2.25 was very close to max size before the upgrade. The Solr upgrade put it slightly over that limit. The Solr dependencies jar is 16.2 MB after the upgrade. The biggest dependencies in that jar are Solr, Lucene, and Zookeeper libraries. The main branch has had a lot removed from it so it did not require this.